packages/hoppscotch-app/components/http/ResponseMeta.vue

```diff
@@ -141,6 +145,36 @@ const props = defineProps<{
   response: HoppRESTResponse
 }>()
 
+const responseSizeTippy = computed(() => {
+  if (
+    props.response.type === "loading" ||
+    props.response.type === "network_fail" ||
+    props.response.type === "script_fail" ||
+    props.response.type === "fail"
+  )
+    return
+  if (Number(props.response.meta.responseSize) <= 1000)
+    return {
+      onShow: () => {
+        return false
+      },
+    }
+  else return { theme: "tooltip", allowHTML: true }
+})
+
+const responseSizeHint = computed(() => {
+  if (
+    props.response.type === "loading" ||
+    props.response.type === "network_fail" ||
+    props.response.type === "script_fail" ||
+    props.response.type === "fail"
+  )
+    return
```

Comment:
Implicit undefined return. Since we return values in other code paths, it is best if you mention the return value here too (implcitly it will be undefined).

```suggestion
    return undefined
```